app/jobs/review_users.py

```diff
@@ -77,10 +78,23 @@ def review():
 
             update_user_status(db, user, status)
             start_user_expire(db, user)
-            report.status_change(user.username, status,
+            bg.add_task(report.status_change, user.username, status,
                                  UserResponse.from_orm(user))
 
             logger.info(f"User \"{user.username}\" status changed to {status}")
 
+        for user in get_users(db, status=UserStatus.expired):
 
-scheduler.add_job(review, 'interval', seconds=10, coalesce=True)
+            active = user.expire and datetime.timestamp(user.expire) >= now.timestamp()
+            if active:
+                status = UserStatus.active
+            else:
+                continue
+
+            update_user_status(db, user, status)
+            xray.operations.add_user(user)
+            
+            logger.info(f"User \"{user.username}\" status fixed.")
```

Comment:
I previously removed this section because I think it must be fixed where it's happening rather than writing an extra check. It rarely happens that a user gets expired by mistake ( never happened to me ), I appreciate any evidence or clue to finding the problem.